cli/CMakeLists.txt

```diff
@@ -15,7 +15,6 @@
 # specific language governing permissions and limitations
 # under the License.
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
As I was checking over the commit, I saw this and thought I added it so that the compiled grpc files would be included. On looking at the history, it was not my addition- I'll put it back.

But it raises the question: why is it there to begin with?